app/Http/Routes/AdminRoutes.php

```diff
@@ -121,6 +121,21 @@ public function map(Registrar $router)
                 $router->delete('{incident_template}/delete', 'IncidentController@deleteTemplateAction');
             });
 
+            $router->group(['prefix' => 'subscribers', 'middleware' => 'app.subscribers'], function ($router) {
```

Comment:
Don't add the `app.subscribers` middleware here. Dashboard should give all access.